### DISCUSSION TITLE
Connect to OBS with different device

### DISCUSSION THREAD

**Discussion by Glycode** [negue/meme-box#410] [Q&A] [answered]
Lets say OBS is open in PC 1
Memebox is open in PC 2

Would it be possible to connect to PC 1's OBS with PC 2's Memebox?

Both devices are in the same network

**Comment by negue** [comment, marked as answer]
Yes, that works. As long your OBS WebSocket isn't blocked by a firewall. Memebox on PC 2 can connect to the OBS PC 1.

I currently don't know if the Dashboard automatically refresh on an OBS Setting Change - but a restart should definitly show if the connection settings worked or not.

**Reply by Glycode** [reply]
sorry for the late response. It works. But only if I start OBS after MemeBox

**Reply by negue** [reply]
no worries!!

the OBS Connection currently is a bit buggy yes - even though I have added 5 connection tries - and scripts should also retry to connect if the WebSocket isn't connected yet 

I currently don't know where / what I should add to improve this kind of state even more 😀  

maybe on the next version when there is a new way to contruct more script "stuff" this could be refactored again and maybe improved as well


the "start OBS after Memebox" I'd have to retest again - I think I had that already tried out -- what is the time difference between OBS start and Memebox start ?

**Reply by Glycode** [reply]
The time difference is somewhat alternating. I sometimes start OBS on my PC then turn on my laptop and start memebox and once I notice it's not connected to OBS I just restart OBS and it usually works after that. So it's not even a minute I think
